rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -1081,11 +1080,18 @@ private BtcTransaction createSvpSpendTransaction(BtcTransaction svpFundTxSigned,
     }
 
     private void addSvpSpendTransactionInputs(BtcTransaction svpSpendTransaction, BtcTransaction svpFundTxSigned, Federation proposedFederation) {
+        Script proposedFederationRedeemScript = proposedFederation.getRedeemScript();
         Script proposedFederationOutputScript = proposedFederation.getP2SHScript();
         addInputFromMatchingOutputScript(svpSpendTransaction, svpFundTxSigned, proposedFederationOutputScript);
-
-        Script flyoverProposedFederationOutputScript = getFlyoverScriptPubKey(bridgeConstants.getProposedFederationFlyoverPrefix(), proposedFederation.getRedeemScript());
-        addInputFromMatchingOutputScript(svpSpendTransaction, svpFundTxSigned, flyoverProposedFederationOutputScript);
+        svpSpendTransaction.getInput(0)
```

Comment:
Should we also update the tests where the spend tx is created? Make sure it has the expected script sig after creation

Reply:
i think thats covered with the test that tests the method, but there's no harm in adding it sure

Reply:
updated